Library/SharedFunctionsTests.swift

```diff
@@ -182,4 +182,62 @@ final class SharedFunctionsTests: TestCase {
     XCTAssertTrue(ksr_isOSVersionAvailable(12.123))
     XCTAssertTrue(ksr_isOSVersionAvailable(12.9))
   }
+
+  func testAttributedStringWithLinks() {
```

Comment:
Should we also add a test that demonstrates inability to identify multiple occurrences of the same link / text?